src/server/runtime/AppsEngineDenoRuntime.ts

```diff
@@ -179,10 +183,58 @@ export class DenoRuntimeSubprocessController extends EventEmitter {
     }
 
     private setupListeners(): void {
-        this.deno.stdout.on('data', this.parseOutput.bind(this));
         this.deno.stderr.on('data', this.parseError.bind(this));
-
         this.on('ready', this.onReady.bind(this));
+
+        let messageBuffer: string[] = [];
+
+        this.deno.stdout.on('data', async (chunk: Buffer) => {
```

Comment:
I had to move this out because I needed the `messageBuffer` to live outside the scope of the callback (previously `parseOutput`).

There is likely a better way to do this, but it can be left to another time maybe